firedrills.md

```diff
@@ -1057,7 +1057,7 @@ node-1$ sudo rm -rf /var/lib/gravity/site/update/gravity
 Now that we’ve learned how to do rollbacks, let’s reattempt the upgrade but this time execute it all the way through successful completion. To get started, let’s launch an upgrade operation in the manual mode. From the upgrade directory, run:
 
 ```bash
-node-1$ sudo ./gravity upgrade --manual
+node-1$ sudo ./upgrade --manual
```

Comment:
I believe it's intention here to use the gravity binary to launch a manual upgrade. This is the same as we document manual upgrades https://gravitational.com/gravity/docs/cluster/#manual-upgrade although our docs aren't the most accurate with the evolution of the UX.